test/drivers/event_class/event_class.h

```diff
@@ -39,15 +39,15 @@ struct fd_poll
 
 struct send_data {
 	int syscall_num;
-	bool greater_snaplen;
-	bool null_sockaddr;
+	bool greater_snaplen = false;
+	bool null_sockaddr = false;
 };
 
 struct recv_data {
 	int syscall_num;
```

Comment:
Why don't we initialize this to `0` as well?

Reply:
This feels like a field that doesn't have a reasonable default (0 will raise an error when reached) so I left it to be explicitly initialized.

Reply:
it was just to avoid specifying it manually to `0` when we use `skip_recv_phase=true`. if `skip_recv_phase==true` no recv syscall will be called so it could be ok to have it to `0`, but i get your point

Reply:
we can do it in a follow up